internal/docker/docker_test.go

```diff
@@ -188,9 +216,14 @@ func TestDockerRun(t *testing.T) {
 				return nil
 			}
 			ctx := t.Context()
-			if err := test.runCommand(ctx, test.docker); err != nil {
-				t.Fatal(err)
+			err := test.runCommand(ctx, test.docker)
+
+			if (err != nil) != test.wantErr {
```

Comment:
nit: Can you break this condition down for readability?

Reply:
I actually copy this from https://github.com/googleapis/librarian/blob/ac2ae0792c0c76ed11eda4b0f9c876dcc002b0b4/internal/librarian/generate_test.go#L132